sdk/servicebus/service.projects

```diff
@@ -0,0 +1,36 @@
+<!--
+  This file allows overrides and extensions to the build configuration defined
+  by the central engineering system, and is considered during pipeline builds for
+  CI, test runs, and package release.
+
+  Based on the SDKType variable (msbuild property or env variable) this will filter
+  the set of projects to to match what type of SDK we are interested in building.
+
+  SDKType values
+   - empty or "client" will build the new track 2 client/data libraries
+   - "mgmtclient" will build the track 2 mgmt libraries
+   - "data" will build the track 1 data plane libraries
+-->
+
+<Project>
+  <ItemGroup Condition="'$(SDKType)' == '' or '$(SDKType)' == 'client'">
+    <ProjectsToInclude Include="$(MSBuildThisFileDirectory)Azure.Messaging.*\**\*.csproj" />
+    <ProjectsToInclude Include="$(MSBuildThisFileDirectory)Microsoft.Azure.WebJobs.Extensions.ServiceBus\**\*.csproj" />
+    <ProjectsToInclude Include="$(MSBuildThisFileDirectory)Azure.ResourceManager.*\**\*.csproj" />
```

Comment:
This doesn't exist today; I added it in to future-proof because this filtering approach isn't widely used and it is likely not obvious for folks to know to add it.  _(where the ci.yml file is standard and contributors should be comfortable extending it as new packages are added)_